microcloud/mdns/mdns.go

```diff
@@ -48,3 +50,18 @@ func dnsTXTSlice(list []byte) []string {
 
 	return parts
 }
+
+// CheckIPv6Status returns true if IPv6 is enabled, false otherwise.
+func CheckIPv6Status() (bool, error) {
+	allIPv6IfacesDisabled, err := os.ReadFile("/proc/sys/net/ipv6/conf/all/disable_ipv6")
+	if err != nil {
+		return false, err
+	}
+
+	newIPv6IfacesDisabled, err := os.ReadFile("/proc/sys/net/ipv6/conf/default/disable_ipv6")
```

Comment:
MicroCloud may select a particular network interface, so I'm not sure if checking `all` or `default` is sufficient. You may need to also check the path for the particular network interface that will be used for the lookup.

Reply:
fixed